zerver/lib/notifications.py

```diff
@@ -335,17 +335,18 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile: UserProfile,
         other_recipients = [r['full_name'] for r in display_recipient
                             if r['id'] != user_profile.id]
         context.update({'group_pm': True})
-        if len(other_recipients) == 2:
-            huddle_display_name = "%s" % (" and ".join(other_recipients))
-            context.update({'huddle_display_name': huddle_display_name})
-        elif len(other_recipients) == 3:
-            huddle_display_name = "%s, %s, and %s" % (
-                other_recipients[0], other_recipients[1], other_recipients[2])
-            context.update({'huddle_display_name': huddle_display_name})
-        else:
-            huddle_display_name = "%s, and %s others" % (
-                ', '.join(other_recipients[:2]), len(other_recipients) - 2)
-            context.update({'huddle_display_name': huddle_display_name})
+        if user_profile.message_content_in_email_notifications:
```

Comment:
Probably we don't need this `if` statement? I think will be cleanest to send all the info to the template, and let the template do the conditionals.

Reply:
Yes, we can do that. But then emails do include/can read all private messages information, which user don't want maybe when they disable this setting.

Reply:
The template will cause the info to be excluded though, right?

Reply:
Yup, template will not show any information even if we pass to template.

Reply:
cool. Then we should just pass everything to the template, and do all the including/excluding in if statements in the template.